html_css/web_accessibility.md

```diff
@@ -0,0 +1,78 @@
+
+3. **Understandable:** Users must be able to understand any information or user interface that is presented to them. For example, if a user tried submitting a form and received an error such as "Error 113: Bad data", they wouldn't be able to understand what the error actually means or how to fix whatever caused the error.
+
+4. **Robust:** Content must be accessible by current assistive technologies and other user agents, and must remain accessible as those technologies advance.
+
+#### Conformance Levels
+
+Conformance levels get mentioned at a couple of other points in these lessons, and we're only mentioning them here to briefly explain what they are ahead of time. To put it simply, the WCAG has three different conformance levels, each of which is made up of various success criteria, or rules, that must be followed in order to successully meet that level of conformance. You don't need to worry about meeting any of these conformance levels completely for the purposes of these lessons, you only need to know that they exist (luckily their naming is pretty easy to remember).
+
+* **Level A**, or essential support, is the minimum level of conformance for the WCAG.
+* **Level AA**, or ideal support, is the level many organizations strive for. Meeting this level also requires meeting Level A.
+* **Level AAA**, or specialized support, isn't recommended for entire sites to meet in full, as some content may make it impossible to meet this conformance level. Meeting this level would require also meeting both Level A and Level AA.
+
+### (Before) Implementing Accessibility
+
+The lessons that follow dive into several concepts that can help you improve the accessibility of your websites, but they don't cover *every* facet of accessibility. The goal of these lessons is just to introduce you to some of the more common or simpler concepts that you should get into the habit of checking for moving forward.
+
+Even though we're only introducing you to some of these a11y concepts, you may still worry that your site isn't accessible enough. The first thing to keep in mind is that **no site will ever be 100% accessible**, so don't try to aim for such an impossible goal. Sometimes the purpose or concept of a site even requires some things to not be accessible in certain ways.
+
+The second thing to keep in mind is that just taking those first few steps towards creating accessible websites matters just as much as the many more steps you'll take in the future. Even if you only add one a11y feature to your websites for now, what you may think is a minor addition could actually be a huge improvement for more users than you realize. So don't feel like you have to make everything perfect or that you need to add everything at once when implementing accessibility, especially when you're just starting out.
+
```

Comment:
lets stick an assignment section at the bottom of this (newly broken out) lesson.

maybe have students check out the WCAG site https://www.w3.org/WAI/standards-guidelines/wcag/
and then link them to something like https://webaim.org/standards/wcag/checklist, which is a much nicer checklist.

You go over all this in later lessons, but being generally familiar with the guidelines and checklists might be helpful.